splice/static/js/controllers/authoring.js

```diff
@@ -98,9 +98,73 @@ angular.module('spliceApp').controller('authoringController', function($controll
      * Send tiles to backend for publication.
      * Assumes data is correct.
      */
+    var cloneTile = function (tile) {
+      if (null === tile || "object" !== typeof tile) return tile;
+
+      var copy = tile.constructor();
+      for (var attr in tile) {
+        if (tile.hasOwnProperty(attr)) {
+          if (attr === "imageURI" || attr === "enhancedImageURI") {
+            copy[attr] = "";
+          } else {
+            copy[attr] = tile[attr];
+          }
+        }
+      }
+      return copy;
+    };
+
+    var compressPayload = function (tiles) {
+      /* *
+       * compress the payload for publishing. Note that the tiles might be cached,
+       * therefore we create a cheap (all imageURIs won't be cloned) copy here
+       */
+      var copies = {};
+      var uri2id = {};
+      var assets = {};
+      var id = 0;
+
+      for (var locale in tiles) {
+        if (tiles.hasOwnProperty(locale)) {
```

Comment:
I think in this use of the data, we don't really care about prototypical inheritance.
Since the object is data (to be serialized) and not code, we won't ever care.

In fact, for this purpose, the aesthetics of this is better:
```javascript
if ("property" in obj) {
}
```

But either property checks are fine.

That said, I'd rather the code be more readable at this point, with less lines, i.e. only having 1 check